frontend/src/modules/entitieslist/actions.js

```diff
@@ -84,13 +92,25 @@ export function get(
 
         const response = await fetch(url, requestParams);
         const content = await response.json();
-        dispatch(receive(content.entities));
+        dispatch(receive(content.entities, content.has_next));
```

Comment:
It's a bit confusing to have `has_next` and `hasMore` - should we unify that?

Reply:
`<InfiniteScroll>` accepts that argument as `hasMore`, which is why I called in that in my code, and the API exposes `has_next`. I don't want to mess with the API as part of Translate.Next (not until we're done with rewriting everything), so we're bound to having this difference. The question is, which term do we think is better, and thus want to use in our code? I don't really have an opinion there, if you have one, please let me know.